.travis.yml

```
@@ -7,12 +7,13 @@ before_install:
   - python .ci/check_travis_tag.py
   - docker build -t aiida_cp2k_test .
   - docker run -d aiida_cp2k_test
+  - sleep 30 # wait untill the container is launched
```

```suggestion
  - sleep 30 # wait until the container is launched
```
at some point we need to find a better solution for this... good for now

---

Ok, and thanks for catching the typo